corso_website_fe/src/Main Page Components/AllServices.tsx

```diff
@@ -41,7 +41,9 @@ function AllServices() {
                             </div>
                             <div className="service-info">
                                 <h2 className="service-name">{service.serviceName}</h2>
-                                {/* <a className="details-hyperlink" href="#">View Details</a> */}
```

Comment:
Is this meant to be commented?

Reply:
i had to comment this out in my previous pr as it was blocking the check. for this pr I added the proper implementation, therefore I removed my comment. so to answer your question, yes it was meant to be commented out